analyzers/its/sources/BlazorSample/BlazorSample/BlazorSample.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
For this project, you can delete everything except:

- WeatherForecast
- WeatherForecastService (Fix indenting)
- _Host
- _Layout
- Counter
- Error
- FetchData
- Index
- MainLayout
- NavMenu
- SurveyPrompt
- _Imports
- App
- Program (remove unused usings)